sunpy/map/mapbase.py

```diff
@@ -2049,7 +2049,11 @@ def plot(self, annotate=True, axes=None, title=True,
                               SunpyUserWarning)
 
         # Normal plot
-        plot_settings = copy.deepcopy(self.plot_settings)
+        try:
+            plot_settings = copy.deepcopy(self.plot_settings)
+        except NotImplementedError:
```

Comment:
I mean this feels like a horrific hack, but as long as we revert it it's better than a load of fails I suppose. Not so sure about backporting this commit though.

Reply:
Fair about the backport but I feel like the rest does need to be backported. I can do it manually?

Reply:
yeah seems good.